src/main/kotlin/com/chattriggers/ctjs/CTJS.kt

```diff
@@ -87,11 +93,15 @@ object CTJS {
 
         registerHooks()
     }
-
-    fun makeWebRequest(url: String): URLConnection = URL(url).openConnection().apply {
-        setRequestProperty("User-Agent", "Mozilla/5.0 (ChatTriggers)")
-        connectTimeout = 3000
-        readTimeout = 3000
+    fun makeWebRequest(url: String): URLConnection {
```

Comment:
Let's pay a bit more attention to the formatting here. Newlines before functions, 4-space indentation, etc.